src/inc/Request.php

```diff
@@ -653,11 +651,10 @@ public function getClientUserAgent()
      */
     public function getConfigValue($key, $default = '')
     {
-        $value = $default;
         if (array_key_exists($key, $this->config)) {
-            $value = $this->config[$key];
+            return $this->config[$key];
```

Comment:
Same here. The "Happy Path" should be the one where we can get a Config Value. So I'd switch that like for ```getParameter```

Reply:
Updated.